File: Practice Arrays/codigo_cs/22Count of Maximum.cs
using System;
using System.Linq;
using System.Collections.Generic;
public class Test22
{
    public static void CountMaximum()
    {
        // your code goes here
        int t = int.Parse(Console.ReadLine());
        for (int _ = 0; _ < t; _++)
        {
            int n = int.Parse(Console.ReadLine());
            int[] a = Console.ReadLine().Split().Select(int.Parse).ToArray();
            Dictionary<int, int> dict = new Dictionary<int, int>();
            int max_count = 0;
            int elem = 0;
            for (int i = 0; i < n; i++)
            {
                if (dict.TryGetValue(a[i], out int value))
                {
                    dict[a[i]] = ++value;
                }
                else
                {
                    dict.Add(a[i], 1);
                }

                if (dict[a[i]] > max_count)
                {
                    max_count = dict[a[i]];
                    elem = a[i];
                }
                else if (dict[a[i]] == max_count && a[i] < elem)
                {
                    elem = a[i];
                }
            }
            Console.WriteLine($"{elem} {max_count}");
        }
    }
}

File: Practice Arrays/22Count of Maximum.py
t=int(input())
for _ in range(t):
    n=int(input())
    arr=list(map(int,input().split()))
    hs={}
    max_count=0
    elem=0
    for num in arr:
        if num in hs:
            hs[num]+=1
        else:
            hs[num]=1
        if hs[num]>max_count:
            max_count=hs[num]
            elem=num
        elif hs[num]==max_count and num<elem:
            elem=num
    print(elem,max_count)